sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -63,7 +63,8 @@ export class CASService {
     sin: string,
     lastName: string,
```

Comment:
The last name uppercase conversion is happening at the consumer end. As we are doing conversions here, as a suggestion we can move the uppercase conversion into this method.

Reply:
I am not sure if I am following. Is the suggestion to move the `toUpperCase` inside the `convertToASCII`?
The `toUpperCase` is a CAS requirement and it is happening right before CAS endpoint is called.

Reply:
My bad, I missed notice the `toUpperCase()` here. My point was to remove the `toUpperCase` at the consumer end. 

![image](https://github.com/user-attachments/assets/9169804b-ce07-458b-a27d-90e4f4a3a361)

Reply:
Ohhh I can see now.
I saw the upper case and "characters conversation" more related to a CAS-specific "payload" preparation.
For instance, doing the `toUpperCase` here would indicate the character's conversation should be happening here also, IMO.

Reply:
Remove the duplicated `toUpperCase` 😉